tests/test_endpoints.py

```diff
@@ -112,3 +112,11 @@ def test_deserialize(self):
 class TestBeatmapsetDiscussionVotes(TestCase):
     def test_deserialize(self):
         api.beatmapset_discussion_votes().votes[0].score
+
+
+class TestCreateNewPM(TestCase):
+    def test_deserialize(self):
+        # Target ID of 2070907 is Tillerino
+        scoped_api = OssapiV2(client_id, client_secret, redirect_uri="http://localhost:9409/",
+                              scopes=["chat.write"], strict=True)
+        scoped_api.create_pm(user_id=2070907, message="Integration test please ignore")
```

Comment:
we don't have a good way to deal with multiple differently scoped clients right now and this breaks in weird ways if we leave it mixed in the tests, I think it's fine to leave out the test for now and we can revisit once we rewrite auth handling